solr/core/ivy.xml

```diff
@@ -61,15 +61,6 @@
     <dependency org="com.fasterxml.jackson.core" name="jackson-annotations" rev="${/com.fasterxml.jackson.core/jackson-annotations}" conf="test"/>
     <dependency org="com.fasterxml.jackson.dataformat" name="jackson-dataformat-smile" rev="${/com.fasterxml.jackson.dataformat/jackson-dataformat-smile}" conf="compile"/>
 
-
-    <dependency org="org.apache.hadoop" name="hadoop-common" rev="${/org.apache.hadoop/hadoop-common}" conf="compile.hadoop"/>
-    <!--
-      hadoop-hdfs, hadoop-annotations and hadoop-auth are runtime dependencies,
-      so even though they are not compile-time dependencies, they are included
-      here as such so that they are included in the runtime distribution.
-     -->
-    <dependency org="org.apache.hadoop" name="hadoop-hdfs" rev="${/org.apache.hadoop/hadoop-hdfs}" conf="compile.hadoop"/>
-    <dependency org="org.apache.hadoop" name="hadoop-annotations" rev="${/org.apache.hadoop/hadoop-annotations}" conf="compile.hadoop"/>
     <dependency org="org.apache.hadoop" name="hadoop-auth" rev="${/org.apache.hadoop/hadoop-auth}" conf="compile.hadoop"/>
```

Comment:
solr.core.src.java.org.apache.solr.security.KerberosFilter uses org.apache.hadoop.security.authentication.server.AuthenticationFilter from hadoop-auth JAR. That class is itself used by KerberosPlugin in the same package